packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/DiamondInheritanceBaseLevelStruct.java

```diff
@@ -0,0 +1,109 @@
+package software.amazon.jsii.tests.calculator;
+
+/**
+ * EXPERIMENTAL
+ */
+@javax.annotation.Generated(value = "jsii-pacmak")
+@software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
+public interface DiamondInheritanceBaseLevelStruct extends software.amazon.jsii.JsiiSerializable {
+    /**
+     * EXPERIMENTAL
+     */
+    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
+    java.lang.String getBaseLevelProperty();
```

Comment:
Can we remove the "get"?

Reply:
Let's discuss. This is a somewhat ambiguous style issue and is complicated by the difference between immutable structs and classes with mutable properties.

Reply:
We resolved this offline. We're going to keep get...() on the struct properties to be consistent with the class implementation where properties can also have setters.